sdk/keyvault/azure.keyvault.secrets.txt

```diff
@@ -0,0 +1,57 @@
+**** Package/Module azure.keyvault.secrets****
+class SecretClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_secret(self, name: str, **kwargs) -> bytes: ...
+	def delete_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_secret(self, name: str, version: str=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
+	def list_deleted_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.DeletedSecret]: ...
+	def list_secret_versions(self, name: str, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def list_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def purge_deleted_secret(self, name: str, **kwargs): ...
+	def recover_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
+	def restore_secret(self, backup: bytes, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
+	def set_secret(self, name: str, value: str, content_type: str=None, enabled: bool=None, not_before: datetime.datetime=None, expires: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
+	def update_secret_properties(self, name: str, version: str=None, content_type: str=None, enabled: bool=None, not_before: datetime.datetime=None, expires: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
```

Comment:
Here I have no problem to leave it as "enabled=None", because it's more explicit that None means "do not change whatever exists" (just follow-up on previous comment)

Reply:
Is it possible to clear any of the values above? If so, how would I do that?

Reply:
No, one can only set values.

Reply:
Then the only other ability would be to reset a value to its default value.

Reply:
We can also move these into kwargs.